Remotion/Web/UnitTests/Core/ExecutionEngine/WxeContextTest.cs

```diff
@@ -38,43 +40,42 @@ public class WxeContextTest
   private Type _functionType;
   private string _functionTypeName;
   private string _resource;
+  private NameValueCollection _queryString;
 
   [SetUp]
   public virtual void SetUp ()
   {
-    NameValueCollection queryString = new NameValueCollection();
-    queryString.Add(WxeHandler.Parameters.ReturnUrl, "/Root.wxe");
+    _queryString = new NameValueCollection();
+    _queryString.Add(WxeHandler.Parameters.ReturnUrl, "/Root.wxe");
 
-    _currentHttpContext = WxeContextMock.CreateHttpContext(queryString);
+    _currentHttpContext = WxeContextMock.CreateHttpContext(_queryString);
 
     _functionType = typeof(TestFunction);
     _functionTypeName = TypeUtility.GetPartialAssemblyQualifiedName(_functionType);
     _resource = "~/Test.wxe";
 
     UrlMappingConfiguration.Current.Mappings.Add(new UrlMappingEntry(_functionType, _resource));
 
-    _currentWxeContext = new WxeContextMock(_currentHttpContext, queryString);
+    _currentWxeContext = new WxeContextMock(_currentHttpContext, _queryString, new WxeUrlSettings(), new WxeLifetimeManagementSettings());
     WxeContext.SetCurrent(_currentWxeContext);
```

Comment:
considering the interdpeendnecies: it's time to make this into a local variable, I think and initialize the WxeContext.SetCurrent in the test (if it's actually required to have a current set.